### PR TITLE
add camel case id lint rule

### PR DESCRIPTION
**Pull request by felixhaeberle**
this PR adds the camel case id lint rule for the example on YT to the marketplace.

### CODE REVIEW

File: inlang/source-code/message-lint-rules/camelCaseId/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# What does this rule do?
+
+Checks for messages to have a snake case message id.
+
+Having the message id's in snake case format is beneficial because it allows for easier integration with other tools and libraries, like `@inlang/paraglide-js`.
```
Comment (flornkm) [inline inlang/source-code/message-lint-rules/camelCaseId/README.md:5]: I think you have copied the `README.md` from the snake case ID. This file has to rename snake case to camel case.

File: inlang/source-code/message-lint-rules/camelCaseId/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```json
+{
+  "$schema": "https://inlang.com/schema/project-settings",
+  "sourceLanguageTag": "en",
+  "languageTags": [
+    "en",
+    "de"
+  ],
+  "modules": [
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-camel-case-id@latest/dist/index.js"
```
Comment (flornkm) [inline inlang/source-code/message-lint-rules/camelCaseId/README.md:20]: I guess this link is available after the merge, so should be fine that I can't see it.

**Review by flornkm — CHANGES_REQUESTED** [review]

**Comment by flornkm** [comment]
@felixhaeberle the code itself looks good to me and I tested it. The only thing is the readme which gets also shown on the marketplace, otherwise you're good to merge 👍

File: inlang/source-code/message-lint-rules/camelCaseId/README.md (hunk 1 above)
Comment (felixhaeberle) [reply inlang/source-code/message-lint-rules/camelCaseId/README.md]: updated ✅

**Pull request merged**